app/controllers/callbacks_controller.rb

```diff
@@ -7,7 +7,7 @@ def github
     auth = request.env['omniauth.auth']
 
     user = User.from_omniauth(auth)
-    user.update(token: auth.credentials.token)
+    user.assign_attributes(token: auth.credentials.token)
```

Comment:
Come on, just `user.token = auth.credentials.token`

Reply:
Also, why not to do it in `from_omniauth`?
Feels completely OK to do that every time you cast omniauth payload into a User record.